astropy/nddata/mixins/tests/test_ndarithmetic.py

```diff
@@ -1,14 +1,14 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 
-
 import pytest
 import numpy as np
 from numpy.testing import assert_array_equal, assert_array_almost_equal
 
-from astropy.nddata.nduncertainty import (StdDevUncertainty, VarianceUncertainty,
-                              InverseVariance,
-                              UnknownUncertainty,
-                              IncompatibleUncertaintiesException)
+from astropy.nddata.nduncertainty import (
+    StdDevUncertainty, VarianceUncertainty,
```

Comment:
could we not do this type of reformatting? it's neither less lines, nor more readable.

Reply:
Emacs was complaining about line too long...

Reply:
I think "line too long" is a valid reason for this kind of reformatting, especially if the names are very long.

Reply:
ok, well, I give in, we can always reformat in the future, or just revise the line lengths to be e.g. 100 🤷‍♀

Reply:
> or just revise the line lengths to be e.g. 100

I was thinking about this, not 100 but 88 which is the magic number used by black. I made some stats and will open an issue ;).

Reply:
I'm not supporting random stuff just because it's supported by black. We did use 80/100/120 characters in the submodules, so should just make up out mind and pick one of those rather than go with something that is known to be messing up formatting on purpose.
(I'm sorry, but black is the worst thing since sliced bread (that was also just invented to spoil the real stuff)